### PR TITLE
FIWARE Lab build script yum repository URL change

### PR DESCRIPTION
**Pull request by fgalan**
@pratid

### CODE REVIEW

File: scripts/build/fiwarelab/build-fiwarelab-image.sh [hunk 1]
```diff
@@ -133,7 +133,7 @@ FIWARE_REPO=$(mktemp)
 cat > $FIWARE_REPO <<EOF
 [testbed-fiware]
 name=Fiware Repository
-baseurl=http://repositories.lab.fiware.org/repo/rpm/x86_64/
+baseurl=http://repositories.lab.fiware.org/repo/rpm/$releasever/x86_64/
```
Comment (pratid) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: Please do not specify architecture, just `http://repositories.lab.fiware.org/repo/rpm/$releasever`

File: scripts/build/fiwarelab/build-fiwarelab-image.sh [hunk 2]
```diff
@@ -133,7 +133,7 @@ FIWARE_REPO=$(mktemp)
 cat > $FIWARE_REPO <<EOF
 [testbed-fiware]
```
Comment (pratid) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: Just an aesthetical issue: we prefer calling it simply `[fiware]`, given that now repositories are not exclusive of the testbed enviroment.

File: scripts/build/fiwarelab/build-fiwarelab-image.sh (hunk 1 above)
Comment (fgalan) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: Fixed in ba0e05a09b7ff34b15467d245287c5c008bb650c

File: scripts/build/fiwarelab/build-fiwarelab-image.sh (hunk 2 above)
Comment (fgalan) [inline scripts/build/fiwarelab/build-fiwarelab-image.sh]: Fixed in b9306e1

**Comment by pratid** [comment]
LGTM

**Pull request merged**